conf/firmwares/subsystems/shared/imu_px4fmu_v2.4.makefile

```diff
@@ -0,0 +1,52 @@
+# Hey Emacs, this is a -*- makefile -*-
+#
+# PX4 Pixhawk IMUconsists of two (internal) IMU's and one internal magneto. Also an optional external magneto
+#
+# L3GD20 +  LSM303D
+
+include $(CFG_SHARED)/spi_master.makefile
+
+IMU_PX4FMU_CFLAGS += -DIMU_TYPE_H=\"imu/imu_px4fmu_v2.4.h\"
+IMU_CFLAGS  = -DIMU_TYPE_H=\"imu/imu_px4fmu_v2.4.h\"
```

Comment:
preferably all these variables have the imu name in them, so like `IMU_PX4FMU_CFLAGS` above...
Or at least the currently unused `IMU_PX4FMU_CFLAGS` above...